EAD-Session-4-Assignment/WebContent/login.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Log In Metacube Parking</title>
+</head>
+<body align="center">
+	<form action="LogInServlet" method="post" onsubmit="return validation(this)">
+		<h1>Welcome To Metacube Parking</h1>
+		<h2>Log In Here</h2>
+		<table align="center" cellpadding = "10%" style="text-align:left">
+			<tr>
+				<td>email :</td>
+				<td><input name="email" type="text" placeholder="email" size="60%" required></td>
```

Comment:
Apply the validation for email id here. If the user enters the wrong email id that does not follow the email id standard or keep it as blank it may throw an error.

Reply:
Done